UACloudLibraryServer/Startup.cs

```diff
@@ -155,7 +155,7 @@ public void ConfigureServices(IServiceCollection services)
 #endif
             }
 
-            var serviceName = Configuration["Application"] ?? "UACloudLibrary";
+            var serviceName = Configuration["Application"] ?? "Opc.Ua.Cloud.Library";
```

Comment:
Not sure if AWS allows dots in the name, this one I would keep as is.

Reply:
I agree. better on the safe side